postgres/changelog.d/17109.changed

```diff
@@ -0,0 +1 @@
+PostgreSQL: Optimise table count query
```

Comment:
can we change the changelog to fixed to avoid major version bump?

Reply:
The behaviour is slightly different, `max_relation` is not used to limit the table count anymore so the reported value may change. Is it alright to have this under a `fixed` changelog?

Reply:
`changed` does make sense. It will be to mention max_relation limit is removed. maybe `Optimise table count query. Removed max_relations table count limit`?